### DISCUSSION TITLE
unable to create desired geometry phase field model

### DISCUSSION THREAD

**Discussion by ankit72242** [idaholab/moose#28939] [Q&A Modules: Phase field]
this geometry i create ![image](https://github.com/user-attachments/assets/51ee2d8b-0bd1-4cb1-bef4-e676bbab9f44)
this is desired 
![image](https://github.com/user-attachments/assets/deb2ca1c-c58f-40f6-a811-1b2d1805ab95)
[testz1 (copy).txt](https://github.com/user-attachments/files/17533535/testz1.copy.txt)
please guide me im using 
type = LatticeSmoothCircleIC so that at interface i want to create circle/square type structure 
can you please guide me

**Comment by GiudGiud** [comment]
Hello

This lattice of circle does not look so regular. For an irregular lattice you can use this object and specify them one by one (you can generate them with code outside of moose though) 

https://mooseframework.inl.gov/source/ics/SpecifiedSmoothCircleIC.html

What does the blue/red region represent? Are all these elements in the same subdomain?

Guillaume

**Comment by ankit72242** [comment]
Two rectangular block represent one phase1
And red region is different phase2
And these green bubble at interface are corresponding to evolution of
different phase3 came out at expense of both phase1+phase2
Basically these small circle are seed of phase 2
Now I want to put these circle separate by certain distance in interface
strip like rectangular region

Aim is...these small circle should grow towards each other in non uniform
way within middle region of domain only.

On Sun, 27 Oct, 2024, 6:51 pm Guillaume Giudicelli, <
***@***.***> wrote:

> Hello
>
> This lattice of circle does not look so regular. For an irregular lattice
> you can use this object and specify them one by one (you can generate them
> with code outside of moose though)
>
> https://mooseframework.inl.gov/source/ics/SpecifiedSmoothCircleIC.html
>
> What does the blue/red region represent? Are all these elements in the
> same subdomain?
>
> Guillaume
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/discussions/28939#discussioncomment-11066140>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBZJNQMS4FNMTTSX2YC4ARLZ5TSEPAVCNFSM6AAAAABQVWDM5KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGYYTIMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 


Disclaimer:- This footer text is to convey that this email is sent by one 
of the users of IITH. So, do not mark it as SPAM.

**Reply by GiudGiud** [reply]
Should the circle be on a regular lattice?

There are different variables to describe the existence of each phase right?
What does your IC block look like right now?
